backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/ForgotPassword.cshtml.cs

```diff
@@ -95,7 +95,7 @@ private async Task SendPasswordResetEmail(string userName, string callbackUrl)
                 Address = Input.Email,
                 PlaceholderContent = new Dictionary<string, string>(),
                 TemplateType = EmailTemplate.ResetPassword,
-                SenderName = "Reset Password",
+                SenderName = "Resetare parolă",
```

Comment:
Shouldn't this be the subject of the email and the sender still be "Echipa Jurnal Medical" ?